docs/governance.rst

```diff
@@ -0,0 +1,253 @@
+Contributor
+  Person listed in the `all-contributors file`_.
+  The community decides on the content of this file using the same process as any
+  other change to the Repository (see below) allowing the meaning of "Contributor"
+  to evolve independently of the Decision-making rules.
+
+Maintainer
+  A Contributor responsible for the long term health of the project and the
+  community. Maintainers have additional rights (see Rules) helping them to
+  resolve conflicts and increase the pace of the development when necessary.
+  Any maintainer can self-remove themselves. Any contributor can become a
+  maintainer by request and with the support of the majority of the current
+  maintainers. Current Maintainers:
+
+  +-----------------------------------+-----------------+
+  | Name                              | Time commitment |
+  +===================================+=================+
+  | `Logan Dowdle`_ (@dowdlelt)       | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Elizabeth DuPre`_ (@emdupre)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Dan Handwerker`_ (@handwerkerd)  | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Ross Markello`_ (@rmarkello)     | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Taylor Salo`_ (@tsalo)           | 3h/week         |
+  +-----------------------------------+-----------------+
+  | `Joshua Teves`_ (@jbteves)        | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Eneko Uruñuela`_ (@eurunuela)    | 0.5h/week       |
+  +-----------------------------------+-----------------+
+  | `Kirstie Whitaker`_ (@KirstieJane)| 0.5h/week       |
+  +-----------------------------------+-----------------+
+
+
+Rules
+`````
+
+1. Potential modifications to the Repository should first be proposed via an 
+   Issue. Rules regarding Votes apply to both Pull Requests and Issues.
+
+   - Every modification of the specification (including a correction of a typo, adding a new Contributor, an extension adding support for a new data type, or others) or proposal to release a new version needs to be done via a Pull Request (PR) to the Repository.
+2. Anyone can open a PR (this action is not limited to Contributors).
+3. PRs adding new Contributors must also add their GitHub names to the 
+   `all-contributors file`_. 
+   This should be done with the allcontributors bot.
+
+   - Contributors may also add themselves to the Zenodo file if they wish, but this is not mandatory.
+4. A PR is eligible to be merged if and only if these conditions are met:
+
+   a) The PR features at least two `Reviews that Approve <https://help.github.com/articles/about-pull-request-reviews/#about-pull-request-reviews>`_
+      the PR from Maintainers of which neither is the author of the PR. 
+      The reviews need to be made after the last commit in the PR (equivalent to 
```

Comment:
I believe we're considering amending this. Thoughts, @tsalo and @emdupre ?

Reply:
Yeah, it gets a bit frustrating when we have "and one more thing..." which I am incredibly guilty of.
The spirit of having two approving reviews is what's important, really -- we don't want someone to have reviewed three months ago, the PR had a lot of changes, and then only one more approval comes in and it's merged. I'm flexible on how to communicate / enforce that, though.

Reply:
Agreed. We should find a way to enforce the spirit of this without having to re-review after fixing typos. In BIDS, they do a pretty good job of keeping approvals after changes without letting major changes slip through, so there must be a way. I say we change it to 

> The reviews need to be made after all major changes have been made in the PR. If new major changes are introduced, reviews must be resubmitted. This does not apply to minor changes, like typo corrections.

And we remove the "discard stale reviews" rule in the repo settings.